### PR TITLE
Fix Docstring Mistake: dataset vs. metric

### LINKED ISSUE #2412: Docstring mistake: dataset vs. metric
This:

https://github.com/huggingface/datasets/blob/d95b95f8cf3cb0cff5f77a675139b584dcfcf719/src/datasets/load.py#L582

Should better be something like:

`a metric identifier on HuggingFace AWS bucket (list all available metrics and ids with ``datasets.list_metrics()``)`

I can provide a PR l8er...

### PR DESCRIPTION
**Pull request by PhilipMay**
PR to fix #2412

### CODE REVIEW

**Comment by PhilipMay** [comment]
IMO this PR is ready for review. I do not know why tests fail...

File: src/datasets/load.py [hunk 1]
```diff
@@ -576,11 +576,11 @@ def load_metric(
...
         path (``str``):
-            path to the dataset processing script with the dataset builder. Can be either:
+            path to the metric processing script with the metric builder. Can be either:
                 - a local path to processing script or the directory containing the script (if the script has the same name as the directory),
-                    e.g. ``'./dataset/squad'`` or ``'./dataset/squad/squad.py'``
-                - a dataset identifier on HuggingFace AWS bucket (list all available datasets and ids with ``datasets.list_datasets()``)
-                    e.g. ``'squad'``, ``'glue'`` or ``'openai/webtext'``
+                    e.g. ``'./metrics/rogue'`` or ``'./metrics/rogue/rogue.py'``
+                - a metric identifier on HuggingFace AWS bucket (list all available metrics with ``datasets.list_metrics()``)
+                    e.g. ``'rogue'`` or ``'bleu'``
```
Comment (lhoestq) [inline src/datasets/load.py:583]:
```suggestion
                    e.g. ``'./metrics/rouge'`` or ``'./metrics/rogue/rouge.py'``
                - a metric identifier on the HuggingFace datasets repo (list all available metrics with ``datasets.list_metrics()``)
                    e.g. ``'rouge'`` or ``'bleu'``
```

**Review by lhoestq — APPROVED** [review]
Thank you !

I just have one comment: we use rouge, not rogue :p

**Comment by lhoestq** [comment]
The CI fail is unrelated to this PR, and it has been fixed on master, merging :)

**Comment by PhilipMay** [comment]
> I just have one comment: we use rouge, not rogue :p

Oops!

**Comment by PhilipMay** [comment]
rebased on master

**Pull request merged**